mods/ra/sequences/structures.yaml

```diff
@@ -438,7 +438,7 @@ sam:
 	muzzle: samfire
 		Length: 18
 		Facings: 8
-		Offset: -1,-2
+		Offset: -1.6,-2
```

Comment:
This should be `-1, 6` (and I mean the whole thing, `-2` needs to be replaced with `6`).